modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/AndesConfigurationManager.java

```diff
@@ -110,8 +111,10 @@
      * Main file path of configuration files into which all other andes-specific config files (if any)
      * should be linked.
      */
-    private static final String ROOT_CONFIG_FILE_PATH = System.getProperty(ServerConstants.CARBON_HOME) +
-            "/repository/conf/";
+    public static final String CARBON_CONFIG_DIR_PATH = "carbon.config.dir.path";
+    public static final String CARBON_HOME = "carbon.home";
+    private static  String ROOT_CONFIG_FILE_PATH;
```

Comment:
Can make ROOT_CONFIG_FILE_PATH a non-static method level variable (given that its now a dynamic value.).